packages/shared/src/consts.ts

```diff
@@ -27,6 +29,8 @@ const MINIMUM_SWAP_AMOUNTS: Partial<
     [Assets.USDC]: '1000000',
     [Assets.BTC]: '390000',
     [Assets.DOT]: '2000000000',
+    [Assets.ARBETH]: '580000000000000', // TODO: ??
```

Comment:
i got the value by querying the partnernet rpc. but this is not deployed to partnernet, right?

Reply:
no its not even on main yet.